tck/api/src/main/java/io/astefanutti/metrics/cdi/se/ConcurrentGaugedConstructorBean.java

```diff
@@ -15,11 +15,11 @@
  */
 package io.astefanutti.metrics.cdi.se;
 
-import org.eclipse.microprofile.metrics.annotation.Counted;
+import org.eclipse.microprofile.metrics.annotation.ConcurrentGauge;
 
-public class CountedConstructorBean {
+public class ConcurrentGaugedConstructorBean {
```

Comment:
Looks like this class (and a number of others) was renamed -- we should keep the old Counter tests working as before

Reply:
Well, the old counters are gone, so I've moved them to Monotonic*Test.
So they are not lost.